app/controllers/documents_controller.rb

```diff
@@ -0,0 +1,69 @@
+class DocumentsController < ApplicationController
+  def index
+      @documents = Document.all.order(created_at: :desc)
+  end
+
+  def new
+    @document = Document.new
+  end
+
+  def create
+    @document = Document.new
+    @document.title = params[:document][:title]
+    @document.description = params[:document][:description]
+    @document.keywords = params[:document][:keywords]
+    @document.user_id = current_user.id
```

Comment:
Consider instantiating the new document off of the current_user object to get this line for free: `current_user.documents.build(params)`